src/rprblender/nodes/blender_nodes.py

```diff
@@ -1148,7 +1148,11 @@ def export(self):
             elif op == 'MODULO':
                 res = self.create_arithmetic(pyrpr.MATERIAL_NODE_OP_MOD, in1, in2)
             else:
-                raise ValueError("Incorrect math operation", op)
+                in3 = self.get_input_value(2)
```

Comment:
Are there other 3 input math operations?

Reply:
Yes, see the description. `WRAP` and `COMPARE` are probably easy to do. But I have no idea what `SMOOTH_MIN` and `SMOOTH_MAX` do exactly.

Reply:
https://www.iquilezles.org/www/articles/smin/smin.htm There's a bunch of diffferent ways to calculate it.  

It looks like blender does:
`ccl_device_inline float smoothminf(float a, float b, float k)
{
  if (k != 0.0f) {
    float h = fmaxf(k - fabsf(a - b), 0.0f) / k;
    return fminf(a, b) - h * h * h * k * (1.0f / 6.0f);
  }
  else {
    return fminf(a, b);
  }
}`